src/math/function1D.cpp

```diff
@@ -354,7 +398,9 @@ EnumOptions<Functions1D::Form> Functions1D::forms()
                                            {Functions1D::Form::LennardJones126, "LennardJones126", 2},
                                            {Functions1D::Form::Buckingham, "Buckingham", 3},
                                            {Functions1D::Form::GaussianPotential, "GaussianPotential", 3},
-                                           {Functions1D::Form::Harmonic, "Harmonic", 1}});
+                                           {Functions1D::Form::Harmonic, "Harmonic", 1},
+                                           {Functions1D::Form::Coulombic, "Coulombic", 2},
+                                           {Functions1D::Form::ShiftedCoulombTruncation, "ShiftedCoulombTruncation", 3}});
```

Comment:
I'd just call this `ShiftedCoulomb`:
```suggestion
                                           {Functions1D::Form::ShiftedCoulomb, "ShiftedCoulomb", 3}});
```